jetty-osgi/test-jetty-osgi/src/test/config/etc/jetty-with-custom-class.xml

```diff
@@ -82,7 +82,7 @@
          <Item>org.eclipse.jetty.plus.webapp.EnvConfiguration</Item>
          <Item>org.eclipse.jetty.webapp.JmxConfiguration</Item>
          <Item>org.eclipse.jetty.websocket.server.JettyWebSocketConfiguration</Item>
-         <Item>org.eclipse.jetty.websocket.jsr356.JavaxWebSocketExtensionConfig</Item>
+         <Item>org.eclipse.jetty.websocket.javax.common.JavaxWebSocketExtensionConfig</Item>
```

Comment:
Ditto.

`JavaxWebSocketExtensionConfig` is not a Jetty `Configuration` class.

Perhaps you want `org.eclipse.jetty.websocket.javax.server.JavaxWebSocketServerConfiguration` ?